app/components/github_integration/webhooks/utils.py

```diff
@@ -127,6 +139,18 @@ async def send_edit_difference(
     )
 
 
+def _shorten_same_repo_links(
+    origin_repo: RepositoryWebhooks, matchobj: re.Match[str]
+) -> str:
+    if (
+        matchobj.group("owner") == origin_repo.owner.name
+        and matchobj.group("repo") == origin_repo.name
+    ):
```

Comment:
This is never true, because `matchobj.group("owner")` includes a trailing slash, so `"ghostty-org/" == "ghostty-org"` fails. A simple change of `+/)` to `+)/` at the end of L27 fixes this.